vs-build/FASTlib/FASTlib.vfproj

```diff
@@ -563,8 +563,39 @@
 			<FileConfiguration Name="Release_Double|x64">
 				<Tool Name="VFCustomBuildTool" CommandLine="CALL ..\RunRegistry.bat Morison" Description="Running Registry for Morison" Outputs="..\..\modules\hydrodyn\src\Morison_Types.f90"/></FileConfiguration>
 			<FileConfiguration Name="Release_Matlab|Win32">
+<<<<<<< HEAD
```

Comment:
This looks like the remains of an unresolved merge conflict. Does this load or run with the `<<<<<<< HEAD` and `>>>>>>> openfast/dev` lines in this file?